lib/src/base-styles/_nx-colors.scss

```diff
@@ -92,6 +94,19 @@
   --nx-color-counter: var(--nx-swatch-grey-05);
   --nx-color-counter-active-text: var(--nx-swatch-white);
 
+  //NxProgressBar
+  --nx-color-progress-bar-text-success: var(--nx-color-alert-success);
+  --nx-color-progress-bar-icon-success: var(--nx-color-alert-success);
+  --nx-color-progress-bar-text-error: var(--nx-color-alert-error);
+  --nx-color-progress-bar-icon-error: var(--nx-color-alert-error);
+  --nx-color-progress-bar-counter-background: var(--nx-color-component-background);
+  --nx-color-progress-bar-counter-border: var(--nx-color-counter);
+  --nx-color-progress-bar-counter-percent: var(--nx-color-counter);
```

Comment:
What are your thoughts on changing this to `--nx-color-progress-bar-counter-text` instead of `percent`? While 'percent' works, I think `text` goes more with how we've named variables up to this point

Reply:
> What are your thoughts on changing this to `--nx-color-progress-bar-counter-text` instead of `percent`? While 'percent' works, I think `text` goes more with how we've named variables up to this point

I think  `--nx-color-progress-bar-counter-text` may be misleading. `--nx-color-progress-bar-counter-percent` colors the content within a counter. Looking at`NxProgressBar`'s source code, I believe this content can only be a percentage and cannot have any text.

I think `--nx-color-progress-bar-counter-number` instead of `--nx-color-progress-bar-counter-percent` could be a good alternative. `number` is easier to understand than `percent`. What do you think?

Reply:
>I think --nx-color-progress-bar-counter-number instead of --nx-color-progress-bar-counter-percent could be a good alternative. number is easier to understand than percent. What do you think?

I'm good with number. A 'meet in the middle' solution!

Reply:
> I think --nx-color-progress-bar-counter-text may be misleading

I think it should be `--nx-color-progress-bar-counter-text`.  Any text within the counter will be this color.  That that text happens to only ever be a percent isn't pertinent to the variable

Reply:
> > I think --nx-color-progress-bar-counter-text may be misleading
> 
> I think it should be `--nx-color-progress-bar-counter-text`. Any text within the counter will be this color. That that text happens to only ever be a percent isn't pertinent to the variable

If by "text" you mean the general content within the counter -- words, numbers, etc... -- then "text" can work. I changed it here: [9b54996](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/9b549969956a09a91e8e6b73ef77002196f2a30a)